nixos/modules/hardware/video/nvidia.nix

```diff
@@ -249,35 +316,31 @@ in
     } ++ singleton {
       name = "nvidia";
       modules = [ nvidia_x11.bin ];
-      display = !offloadCfg.enable;
-      deviceSection = optionalString primeEnabled
-        ''
-          BusID "${pCfg.nvidiaBusId}"
-          ${optionalString syncCfg.allowExternalGpu "Option \"AllowExternalGpus\""}
-          ${optionalString cfg.powerManagement.finegrained "Option \"NVreg_DynamicPowerManagement=0x02\""}
-        '';
-      screenSection =
-        ''
-          Option "RandRRotation" "on"
```

Comment:
This option is for rotating the screen and may have been included by accident.  It is ignored anyway.